user_guide_src/source/libraries/email.rst

```diff
@@ -113,35 +113,39 @@ sending email.
                                                                      set this to an empty string (``''``). See also
                                                                      :ref:`email-ssl-tls-for-smtp`.
-**wordWrap**        true                ``true``/``false`` (boolean) Enable word-wrap.
-**wrapChars**       76                                               Character count to wrap at.
-**mailType**        text                ``text`` or ``html``         Type of mail. If you send HTML email you must send it as a complete web
+**$wordWrap**       true                ``true``/``false``           Enable word-wrap.
+**$wrapChars**      76                                               Character count to wrap at.
+**$mailType**       text                ``text`` or ``html``         Type of mail. If you send HTML email you must send it as a complete web
                                                                      page. Make sure you don't have any relative links or relative image
                                                                      paths otherwise they will not work.
-**charset**         utf-8                                            Character set (``utf-8``, ``iso-8859-1``, etc.).
-**validate**        true                ``true``/``false`` (boolean) Whether to validate the email address.
-**priority**        3                   1, 2, 3, 4, 5                Email Priority. ``1`` = highest. ``5`` = lowest. ``3`` = normal.
-**CRLF**            \\n                 ``\r\n`` or ``\n`` or ``\r`` Newline character. (Use ``\r\n`` to comply with RFC 822).
-**newline**         \\n                 ``\r\n`` or ``\n`` or ``\r`` Newline character. (Use ``\r\n`` to comply with RFC 822).
-**BCCBatchMode**    false               ``true``/``false`` (boolean) Enable BCC Batch Mode.
-**BCCBatchSize**    200                 None                         Number of emails in each BCC batch.
-**DSN**             false               ``true``/``false`` (boolean) Enable notify message from server.
+**$charset**        UTF-8                                            Character set (``utf-8``, ``iso-8859-1``, etc.).
+**$validate**       true                ``true``/``false``           Whether to validate the email address.
+**$priority**       3                   1, 2, 3, 4, 5                Email Priority. ``1`` = highest. ``5`` = lowest. ``3`` = normal.
+**$CRLF**           \\r\\n              ``\r\n``, ``\n`` or ``\r``   Newline character. (Use ``\r\n`` to comply with RFC 822).
+**$newline**        \\r\\n              ``\r\n``, ``\n`` or ``\r``   Newline character. (Use ``\r\n`` to comply with RFC 822).
+**$BCCBatchMode**   false               ``true``/``false``           Enable BCC Batch Mode.
+**$BCCBatchSize**   200                                              Number of emails in each BCC batch.
+**$DSN**            false               ``true``/``false``           Enable notify message from server.
```

Comment:
Please remove all `$` marks in `Preference`.
Because these items may be keys in an array.